### DISCUSSION TITLE
Sequencing with the Jetson Orin

### DISCUSSION THREAD

**Discussion by william-simon** [sirselim/jetson_nanopore_sequencing#28] [General]
Very interesting work, and thank-your for the extensive notes on your setup! 

I was wondering, did you ever work with the Jetson Orin? I would be curious to see if the extra power translates to improved throughput, or if there is a bottleneck in for example the decoding step which reduces the benefit of the increased TOPS.

**Comment by sirselim** [comment]
Hi @william-simon - thanks for reaching out and the kind words.

I didn't develop for the Jetson Orin as it required a special build of Guppy for the ARM + Ampere architecture that is Orin. This can only be done from source code.

The good news is that Dorado is the new production basecaller and it is being built with ARM support from the get go. It's already working really nicely on Orin. Within the next month or so Dorado will replace Guppy in MinKNOW, so with a little effort there is no reason the whole software stack won't run on Orin based Jetson boards.

I expect them to be much better than the previous generation Jetsons, even the Jetson Orin could be an interesting little device to try out.

**Comment by sirselim** [comment]
@william-simon as of today's release for the Mk1c Dorado is now integrated into MinKNOW. This should mean that everything works on Orin devices, but obviously nothing is certain.

Full release notes can be found here: https://community.nanoporetech.com/posts/software-release-23-07-12
